iosApp/iosApp/Info.plist

```diff
@@ -20,8 +20,8 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
-	<key>CADisableMinimumFrameDurationOnPhone</key>
-	<true/>
```

Comment:
oops, fixed in https://github.com/maplibre/maplibre-compose/commit/325d17a2baf8ad2cdf6af7f1e6a8a9fd95787c06